.github/workflows/ci-release.yml

```diff
@@ -8,7 +8,7 @@ on:
       ref:
         required: true
         type: string
-        default: latest
```

Comment:
Looks like a commit from another branch snuck in here. You'll probably want to start w/ a fresh branch off of `latest`, or rebase this, or drop this commit from your branch.